WORKSPACE

```diff
@@ -188,799 +171,325 @@ container_pull(
-        "https://storage.googleapis.com/builddeps/0e24ffdcda4efc7445834c86d29d7416426bafa9073259cc31bcb1f03fb5eaeb",
-    ],
+    name = "libseccomp-aarch64",
+    urls = ["http://mirror.stream.centos.org/9-stream/BaseOS/aarch64/os/Packages/libseccomp-2.5.0-6.el9.aarch64.rpm"],
+    sha256 = "d7dacf315d019b53dcb05a4b07337968c68ad9b6cb056ba89ead2d699cf13fea",
 )
 
 http_file(
-    name = "liburing",
-    sha256 = "049778a480dd02774934b37c127b345d8748bfbec1e584f9c412a41af34eaf89",
-    urls = ["https://storage.googleapis.com/builddeps/049778a480dd02774934b37c127b345d8748bfbec1e584f9c412a41af34eaf89"],
+    name = "systemd-libs",
+    urls = ["http://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os/Packages/systemd-libs-249-9.el9.x86_64.rpm"],
+    sha256 = "708fbc3c7fd77a21e0b391e2a80d5c344962de9865e79514b2c89210ef06ba39",
 )
 
 http_file(
-    name = "liburing-aarch64",
-    sha256 = "253d0c1dc3180f44766c298ae4cd3426ec7a60a41ea0dc50d0884928b031c1b7",
-    urls = [
-        "http://download.fedoraproject.org/pub/fedora/linux/updates/33/Everything/aarch64/Packages/l/liburing-0.7-3.fc33.aarch64.rpm",
-        "https://storage.googleapis.com/builddeps/253d0c1dc3180f44766c298ae4cd3426ec7a60a41ea0dc50d0884928b031c1b7",
-    ],
+    name = "systemd-libs-aarch64",
+    urls = ["http://mirror.stream.centos.org/9-stream/BaseOS/aarch64/os/Packages/systemd-libs-249-9.el9.aarch64.rpm"],
+    sha256 = "683ca2ab7f0aa82baa63fdd248a2cdc13e1dd7ca55294f93e10971c7176ac85d",
 )
 
 http_file(
-    name = "libseccomp",
-    sha256 = "964e39835b59c76b7eb3f78c460bfc6e7acfb0c40b901775c7e8a7204537f8a7",
-    urls = ["https://storage.googleapis.com/builddeps/964e39835b59c76b7eb3f78c460bfc6e7acfb0c40b901775c7e8a7204537f8a7"],
+    name = "liburing",
+    urls = ["http://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/Packages/liburing-0.7-7.el9.x86_64.rpm"],
+    sha256 = "a9d32f2a52149dfc94e14e0519ebec709b1607955e57f4b4604807164f0e3850",
 )
 
 http_file(
-    name = "libseccomp-aarch64",
-    sha256 = "ab5a824d402c717bfe8e01cfb216a70fd4a7e1d66d2d7baa80ac6ad6581081c9",
-    urls = [
-        "http://download.fedoraproject.org/pub/fedora/linux/releases/33/Everything/aarch64/os/Packages/l/libseccomp-2.5.0-3.fc33.aarch64.rpm",
-        "https://storage.googleapis.com/builddeps/ab5a824d402c717bfe8e01cfb216a70fd4a7e1d66d2d7baa80ac6ad6581081c9",
-    ],
+    name = "liburing-aarch64",
+    urls = ["http://mirror.stream.centos.org/9-stream/AppStream/aarch64/os/Packages/liburing-0.7-7.el9.aarch64.rpm"],
+    sha256 = "d42dd2af61c68f2eed3cb4f1cf5af11e2a3b09a1816709b063f5d2c6377a637d",
 )
 
 #imageio rpms and dependencies
 http_file(
     name = "ovirt-imageio-client",
-    sha256 = "a012d8204098d992099620de1d14b423bfd179b54a618deec98cdcb37027a0cd",
-    urls = ["https://storage.googleapis.com/builddeps/a012d8204098d992099620de1d14b423bfd179b54a618deec98cdcb37027a0cd"],
+    sha256 = "3b230c1b6809be7d19b2abfb37bc6b44114c149692d5c4f53ade50981243ef66",
```

Comment:
looks like this is the wrong sha? Should be `f6a31d2076801da21b50b9c75b0cd42d3152aa94e85143dfcfb8e3502e6afa75`

Reply:
Updated some checksums. Borrowed the test from #2170 since it helped me find a few more that were wrong. Oops.